renderer/pages/editor.js

```diff
@@ -1,10 +1,107 @@
+import path from 'path';
 import React from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+import TrafficLights from '../components/traffic-lights';
+import Options from '../components/editor/options';
+import Video from '../components/editor/video';
+
+class Editor extends React.Component {
```

Comment:
Move this class to a file under `/components/editor/index.js` and `import Editor from '/components/editor'`